### PR TITLE
add tracking for the explainer links

### PR DESCRIPTION
**Pull request by bjfletcher**

### CODE REVIEW

File: views/partials/explainer.html [hunk 1]
```diff
@@ -10,13 +10,13 @@ <h2 class="explainer__headline">About Special Reports</h2>
 	</div>
 	<div class="explainer__actions">
 		<div class="explainer__links">
```
Comment (adambraimbridge) [inline views/partials/explainer.html]: `<div class="explainer__links" data-trackable="links">`

File: views/partials/explainer.html [hunk 2]
```diff
@@ -10,13 +10,13 @@ <h2 class="explainer__headline">About Special Reports</h2>
...
-			<a href="http://fttoolkit.co.uk/d/calendar/" class="explainer__link">Future and planned reports</a>
-			<a href="http://www.ft.com/reports-library?ft_site=falcon&desktop=true" class="explainer__link">Reports library</a>
+			<a href="http://fttoolkit.co.uk/d/calendar/" class="explainer__link" data-trackable="plans">Future and planned reports</a>
+			<a href="http://www.ft.com/reports-library?ft_site=falcon&desktop=true" class="explainer__link" data-trackable="library">Reports library</a>
 		</div>
 		<div class="explainer__social-links">
```
Comment (adambraimbridge) [inline views/partials/explainer.html]: `<div class="explainer__social-links" data-trackable="social-links">`

File: views/partials/explainer.html [hunk 3]
```diff
@@ -10,13 +10,13 @@ <h2 class="explainer__headline">About Special Reports</h2>
 	</div>
 	<div class="explainer__actions">
 		<div class="explainer__links">
-			<a href="http://fttoolkit.co.uk/d/calendar/" class="explainer__link">Future and planned reports</a>
-			<a href="http://www.ft.com/reports-library?ft_site=falcon&desktop=true" class="explainer__link">Reports library</a>
+			<a href="http://fttoolkit.co.uk/d/calendar/" class="explainer__link" data-trackable="plans">Future and planned reports</a>
```
Comment (adambraimbridge) [inline views/partials/explainer.html]: `<a href="http://fttoolkit.co.uk/d/calendar/" class="explainer__link" data-trackable="link">Future and planned reports</a>`

File: views/partials/explainer.html [hunk 4]
```diff
@@ -10,13 +10,13 @@ <h2 class="explainer__headline">About Special Reports</h2>
...
+			<a href="http://www.ft.com/reports-library?ft_site=falcon&desktop=true" class="explainer__link" data-trackable="library">Reports library</a>
```
Comment (adambraimbridge) [inline views/partials/explainer.html]: `<a href="http://www.ft.com/reports-library?ft_site=falcon&desktop=true" class="explainer__link" data-trackable="link">Reports library</a>`

File: views/partials/explainer.html [hunk 5]
```diff
@@ -10,13 +10,13 @@ <h2 class="explainer__headline">About Special Reports</h2>
...
 		</div>
 		<div class="explainer__social-links">
 			<span class="explainer__social-links-note">Follow special reports</span>
-			<a class="explainer__twitter o-share__action o-share__action--twitter" target="_blank" href="http://twitter.com/ftreports"><i>Twitter</i></a>
-			<a class="explainer__facebook o-share__action o-share__action--facebook" target="_blank" href="http://www.facebook.com/financialtimes"><i>Facebook</i></a>
+			<a class="explainer__twitter o-share__action o-share__action--twitter" target="_blank" href="http://twitter.com/ftreports" data-trackable="twitter"><i>Twitter</i></a>
```
Comment (adambraimbridge) [inline views/partials/explainer.html]: `<a class="explainer__twitter o-share__action o-share__action--twitter" target="_blank" href="http://twitter.com/ftreports" data-trackable="link"><i>Twitter</i></a>`

File: views/partials/explainer.html [hunk 6]
```diff
@@ -10,13 +10,13 @@ <h2 class="explainer__headline">About Special Reports</h2>
...
+			<a class="explainer__facebook o-share__action o-share__action--facebook" target="_blank" href="http://www.facebook.com/financialtimes" data-trackable="facebook"><i>Facebook</i></a>
```
Comment (adambraimbridge) [inline views/partials/explainer.html]: `<a class="explainer__facebook o-share__action o-share__action--facebook" target="_blank" href="http://www.facebook.com/financialtimes" data-trackable="link"><i>Facebook</i></a>`

**Comment by adambraimbridge** [comment]
Example:

image

Above: The `data-trackable` is not "Luxury 360", as you might expect. Rather, it's `primary-tag`, which allows us to target clicks of all `primary-tag` elements; and importantly, compare `primary-tag` elements in _conatiner A_ with those of _container B_

File: views/partials/explainer.html (hunk 6 above)
Comment (ironsidevsquincy) [inline views/partials/explainer.html]: how do we differentiate between which link is clicked if they're all called `link`?

**Comment by ironsidevsquincy** [comment]
Wont it always be a link though? The problem with using the link's text for reporting is that could easily change (e.g. 'Twitter' => 'Tweet')

**Comment by adambraimbridge** [comment]
Yeah, this is a bad example. I can't imagine why anyone would want to target that link specifically.

**Comment by adambraimbridge** [comment]
Let's move the discussion to https://docs.google.com/document/d/148tz76eji1KdHyqoiX7Y-NMHEA5l8AAe2me8eM_rZ64/ :)

**Pull request merged**